NorthstarDedicatedTest/serverauthentication.cpp

```diff
@@ -388,7 +388,7 @@ bool CBaseClient__ConnectHook(void* self, char* name, __int64 netchan_ptr_arg, c
 	if (!ret)
 		return ret;
 
-	if (!g_ServerBanSystem->IsUIDAllowed(nextPlayerUid))
+	if (IsDedicated() && !g_ServerBanSystem->IsUIDAllowed(nextPlayerUid))
```

Comment:
Looking at this again I think this breaks banlist functionality completely for listen server ^^